classes/pdf/PDF.php

```diff
@@ -145,7 +169,7 @@ public function render($display = true)
     /**
      * Get correct PDF template classes.
      *
-     * @param mixed $object
+     * @param ObjectModel $object
```

Comment:
We can't change here because the signature indicated `mixed` not ObjectModel

Reply:
If it's not an ObjectModel, it will fail...

Reply:
So you need to change the method signature too